### PR TITLE
GetUnspents to use outputHasRunes method

### PR DESCRIPTION
**Pull request by dcorral**
Put outputHasRunes() in a utils file to reuse in GetUnspents

### CODE REVIEW

File: src/Methods/Address/GetUnspents.ts [hunk 1]
```diff
@@ -90,7 +92,7 @@ export default method({
       const [inscriptions, ordinals, runeBalances] = await Promise.all([
```
Comment (danadi7) [inline src/Methods/Address/GetUnspents.ts:92]: Rename `runeBalances` to `hasRunes` for clarity and consistency with `GetSpendables`

File: src/Methods/Address/GetUnspents.ts [hunk 2]
```diff
@@ -87,18 +89,18 @@ export default method({
...
         ord.getOrdinals(outpoint),
-        ord.getRuneOutputsBalancesByOutpoint(outpoint),
+        outputHasRunes(outpoint),
       ]);
 
       // DEPRECATED, REMOVE WHEN CLIENT CATCHES UP
       utxo.ordinals = ordinals;
       utxo.inscriptions = inscriptions;
-      utxo.runes = runeBalances;
+      utxo.runes = hasRunes;
```
Comment (danadi7) [inline src/Methods/Address/GetUnspents.ts:101]: I think this was meant to return the rune balances on the UTXO. Can we fix this on PR68?

**Review by danadi7 — APPROVED** [review]
LGTM!

File: src/Methods/Address/GetUnspents.ts (hunk 2 above)
Comment (danadi7) [reply src/Methods/Address/GetUnspents.ts:101]: Since it's not being used for now I'll merge this PR first.

Comment (dcorral) [reply src/Methods/Address/GetUnspents.ts:101]: I'll add it to PR68

**Pull request merged**